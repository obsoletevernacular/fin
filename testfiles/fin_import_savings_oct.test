Transaction ID,Posting Date,Effective Date,Transaction Type,Amount,Check Number,Reference Number,Description,Transaction Category,Type,Balance
"20171031 111111 1,111 111,111,111,111",10/31/17,10/31/17,Credit,0.84,,10,Credit Dividend,,,109.84
"20171031 111111 1,111 111,111,111,112",10/31/17,10/31/17,Debit,-15,,11,Withdrawal TYPE from ACCT_NO ACCT_TYPE,,Withdrawal,94.84
"20171028 111111 1,111 111,111,111,113",10/28/17,10/28/17,Debit,-20,,12,ATM Withdrawal PLACE   LOCATION,,Debit Card,74.84
"20171020 111111 1,111 111,111,111,114",10/20/17,10/20/17,Credit,40,,13,ACH Deposit ACCOUNT - COMMENT,,ACH,114.84
"20171005 111111 1,111 111,111,111,115",10/5/17,10/6/17,Credit,20,,14,ACH Deposit ACCOUNT - COMMENT,,ACH,134.84
"20171005 111111 1,111 111,111,111,116",10/5/17,10/6/17,Debit,-36.59,,15,ATM Withdrawal PLACE     LOCATION,,Debit Card,98.25
"20171005 111111 1,111 111,111,111,117",10/5/17,10/6/17,Credit,-20,,16,ATM Deposit PLACE     LOCATION,,Debit Card,78.25