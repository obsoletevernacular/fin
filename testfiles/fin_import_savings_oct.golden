1 files processed and stored to .default.obj
summary
    transactions: 7
    transactions_in: 3
    transactions_out: 4
    sum: -30.75
    sum_in: 60.84
    sum_out: -91.59
    avg: -4.39
